src/TimeoutMigrationTool.Raven3.AcceptanceTests/TimeoutMigrationTool.Raven3.AcceptanceTests.csproj

```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <RootNamespace>TimeoutMigrationTool.Raven3.AcceptanceTests</RootNamespace>
```

Comment:
does that have to be specified? Given the project name, I'd expect that to be the default already?

Reply:
Fixed